configuration/ageout.md

```diff
@@ -133,6 +133,14 @@ Storage-based constraints are not an instant hard limit.  Be sure to leave a lit
 
 ## Storage Available-Based Ageout Rules
 
+```{attention}
+Be careful when configuring storage available-based ageout rules. These rules apply to the available storage of the disk partition the well is configured on. If disk space is used by another data source, or even a process outside of Gravwell, the rules will still be enforced. This means that external data can cause data deletion when using this option.
+```
+
+```{attention}
+Be careful when configuring storage available-based ageout rules with multiple wells. If multiple wells are configured on the same disk partition, it's possible to have well configurations "compete" for available disk space. This is especially true if two wells have different storage available-based ageout rules. This means that wells on the same with that use this option can cause each other to delete data.
```

Comment:
Needs revision: "This means **wells on the same with that use** this option..."
Should be "wells on the same partition that use this option"?